core/src/main/resources/hudson/triggers/Messages_de.properties

```diff
@@ -25,5 +25,8 @@ SCMTrigger.getDisplayName={0} Abfrage-Protokoll
 SCMTrigger.BuildAction.DisplayName=Abfrage-Protokoll
 SCMTrigger.SCMTriggerCause.ShortDescription=Build wurde durch eine SCM-�nderung ausgel�st.
 TimerTrigger.DisplayName=Builds zeitgesteuert starten
+TimerTrigger.MissingWhitespace=Es scheinen Leerzeichen zwischen * und * zu fehlen.
+TimerTrigger.no_schedules_so_will_never_run=Keine zeitgesteuerten Builds, somit wird Job nie laufen
 TimerTrigger.TimerTriggerCause.ShortDescription=Build wurde zeitgesteuert ausgel�st.
+TimerTrigger.would_last_have_run_at_would_next_run_at=Letzter Lauf am {0}; N�chster Lauf am {1}.
```

Comment:
Concerned about the syntax highlighting at GitHub, does the semicolon need to be escaped?